src/integrations/remote_integration.rs

```diff
@@ -0,0 +1,8 @@
+use RemoteStatus;
+
+pub trait RemoteIntegration {
+    fn get_status(&mut self) -> RemoteStatus;
```

Comment:
The only reason that I'm passing in a mutable reference to `self` here, is that the Unity integration needs to mutate itself, to track `last_tick` and `last_status`. Otherwise, this could take an immutable reference, i.e. `&self`. Would be nice to find a way to make this immutable again.